packages/view-types/statistical-plots/src/CellSetExpressionPlotSubscriber.js

```diff
@@ -223,10 +225,11 @@ export function CellSetExpressionPlotSubscriber(props) {
     o => o.value === featureValueTransform,
   )?.name;
 
-
   return (
```

Comment:
```suggestion
  // Use empty string when firstGeneSelected is null
  const titleSuffix = firstGeneSelected ? ` (${firstGeneSelected})` : '';
  return (
```

We should keep the ternary part in case `firstGeneSelected` is `null` to avoid displaying ` (null)`